frontend/appflowy_flutter/lib/workspace/presentation/settings/widgets/settings_appearance/create_file_setting.dart

```diff
@@ -0,0 +1,34 @@
+import 'package:appflowy/workspace/application/settings/prelude.dart';
+import 'package:appflowy/workspace/presentation/settings/widgets/settings_appearance/theme_setting_entry_template.dart';
+import 'package:flutter/material.dart';
+import 'package:flutter_bloc/flutter_bloc.dart';
+
+class CreateFileSettings extends StatelessWidget {
+  CreateFileSettings({
```

Comment:
```suggestion
  const CreateFileSettings({
```

Reply:
the const constructor can't have a body.